docs/gatsby-config.js

```diff
@@ -0,0 +1,31 @@
+const themeOptions = require('gatsby-theme-apollo-docs/theme-options');
+
+module.exports = {
+  pathPrefix: '/docs/apollo-cli',
+  plugins: [
+    {
+      resolve: 'gatsby-theme-apollo-docs',
+      options: {
+        ...themeOptions,
+        root: __dirname,
+        subtitle: 'Apollo CLI',
+        description: 'A guide to using Apollo CLI',
+        githubRepo: 'apollographql/apollo-cli',
+        sidebarCategories: {
+          null: ['getting-started'],
+          Contributing: [
+            'contributing/index',
+            'contributing/prerequisites',
+            'contributing/project-structure',
+          ],
+          Configuring: [
+            'usage/config/index',
+            'usage/config/authentication',
+            'usage/config/environment-variables',
+            'usage/config/profiles',
+          ],
+        },
```

Comment:
This is where we setup what the sidebar looks like. `getting-started` is by itself, and the others are under `Contributing` and `Configuring` headings. The header info in the individual docs are responsible for the display title of each item